dlme_airflow/drivers/oai_xml.py

```diff
@@ -1,11 +1,18 @@
-import re
 import logging
 import intake
 import pandas as pd
 
 from lxml import etree
 from sickle import Sickle
 
+# xml namespaces and the prefixes that are used in parsing
+
+NS = {
+    "oai_dc": "http://www.openarchives.org/OAI/2.0/oai_dc/",
+    "mods": "http://www.loc.gov/mods/v3",
+    "marc21": "http://www.loc.gov/MARC21/slim",
+}
+
```

Comment:
I moved NS up to the module level so it could be used in `_get_tag()` without needing to pass it around. It doesn't change.